### ISSUE TITLE
rename-sequence command

### ISSUE THREAD

**Issue by bookdude13** [ProtonLights/cli#32]
Implement command to rename a sequence.
`rename-sequence <admin-key> <name> <new-name>`

**Comment by rjayatilleka** [comment]
I don't think this is necessary. It's just unnecessary work, probably. Or I guess, leave the issue open, but don't implement it. There are more important things.

**Comment by bookdude13** [comment]
Fair enough. If we feel we want/need it, then it should be a quick add.
